[Documentation] Fix typos and add homepage section (#1387)
GeotrekCE/Geotrek-rando-v3: docs/customization/customization-mediamanagement.md

It provides 6 default images that can be customized:

- `android-icon.png`: It has to be 144x144px and controls the icon appearing on Android phones

> - `maskable-icon.png`: It hass to be 144x144px and will be used only on android phones enabling round icons (it will hence be cropped to fit a circle by the android phones)

Reviewer: ```suggestion
- `maskable-icon.png`: It has to be 144x144px and will be used only on android phones enabling round icons (so it will be cropped to fit in a circle by Android phones)
```
